react-components/src/architecture/concrete/pointsOfInterest/CreatePointsOfInterestWithDescriptionCommand.ts

```diff
@@ -49,7 +59,12 @@ export class CreatePointsOfInterestWithDescriptionCommand extends BaseInputComma
     }
 
     const poi = domainObject.addPendingPointsOfInterest(
-      createPointsOfInterestPropertiesFromPointAndTitle(this._point, this._scene, this._content)
+      createPointsOfInterestPropertiesFromPointAndTitle(
+        this._point,
+        this._scene,
+        this._content,
```

Comment:
Just double check if the content here will be a list because of the implementation using the CustomBaseInputCommand

Reply:
Ah, right, we probably have some merge conflicts, but should be easy enough to fix